misaligned.js

```diff
@@ -1,16 +1,39 @@
-const {expect} = require('chai')
-
-function print_color_map() {
+function color_map() {
     const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
     const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
+    let colorCodes=[];
     for (let i = 0; i < majorColors.length; i++) {
         for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
+            //to seperate the formatting from console.log
+            colorCodes.push(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
         }
     }
-    return majorColors.length * minorColors.length;
+    return colorCodes;
+}
+
+function print_color_map(colorCodes) {

```

Comment:
The only purpose of this function is to replace the console.log. What do we gain by it?
I thought it probably enables us to test the functionality in a broader sense, but the test is still checking the output given by `color_map` only, but through `pring_color_map`. Hence my question.